masonry/src/convert_winit_event.rs

```diff
@@ -115,9 +114,11 @@ pub(crate) fn winit_physical_key_to_kbt_code(physical_key: WinitPhysicalKey) ->
         WinitPhysicalKey::Unidentified(_) => Code::Unidentified,
         WinitPhysicalKey::Code(key_code) => match key_code {
             // Variants that don't match 1:1
+            // With winit 0.31, these are renamed to swap the meta and super to match the W3C specs.
+            #[expect(deprecated, reason = "Map even the deprecated keys.")]
             WinitKeyCode::Meta => Code::Super,
-            WinitKeyCode::SuperLeft => Code::Super,
-            WinitKeyCode::SuperRight => Code::Super,
+            WinitKeyCode::SuperLeft => Code::MetaLeft,
+            WinitKeyCode::SuperRight => Code::MetaRight,
```

Comment:
@madsmtm This part is the main part that I'm sure about here.

Reply:
I think it's correct. Note that `WinitKeyCode::Meta` isn't actually emitted in Winit v0.30.x as far as I know.

See also https://github.com/rust-windowing/winit/pull/4018.